src/collar/oc_core.lsl

```diff
@@ -322,7 +322,7 @@ UserCommand(integer iNum, string sStr, key kID) {
             llMessageLinked(LINK_SET, LM_SETTING_SAVE, "global_locked="+(string)g_iLocked,"");
             llMessageLinked(LINK_SET, NOTIFY, "1%WEARERNAME%'s collar has been locked", kID);
             llPlaySound(g_sLockSound,1);
-        } else if(llToLower(sChangetype) == "unlock" && (iNum == CMD_OWNER || iNum == CMD_TRUSTED) && !g_iWelded){
+        } else if(llToLower(sChangetype) == "unlock" && iNum == CMD_OWNER && !g_iWelded){
```

Comment:
Is trusted able to lock the collar?  We wouldn't want someone able to lock but not unlock, so you might need a corresponding change on that side.

Reply:
> Is trusted able to lock the collar? We wouldn't want someone able to lock but not unlock, so you might need a corresponding change on that side.

Trusted has never had access to lock the collar as far as I know. I think the change was an ill-advised response to someone getting trapped in furniture, hoping a Trusted could get them out of the trap when their Owner was off line. Bottom line is a Trusted should not be able to undo something an Owner set. And since the Trusted can not Lock the collar they should not be able to Unlock it.